samples/data-grid/basic/grouped-headers/demo.js

```diff
@@ -1,9 +1,9 @@
 DataGrid.dataGrid('container', {
-    table: {
+    dataTable: {
         columns: {
             id: ['1', '2', '3', '4'],
             product: ['Apples', 'Pears', 'Plums', 'Bananas'],
-            weight: [100, 40, 0.5, 200, 100, 40, 0.5, 200],
+            weight: [100, 40, 0.5, 200],
```

Comment:
Will do full review tomorrow, but quick question: did we fix the original problem here, or just changed the dataset? IMO we should fix he issue where we include disabled rows with superfluous items when rendering the table. Leads to empty rows, and we shouldn't let unused data influence rendering.

Reply:
I just changed the dataset.

The problem with rows is a separate issue and should not be a part of header. I suggest to create a new ticket in Asana.

Reply:
OK, done.